README.md

```diff
@@ -34,6 +34,13 @@ The image recognizes following environment variables that you can set during ini
 |  `MYSQL_DATABASE`      | Database name                             |
 |  `MYSQL_ROOT_PASSWORD` | Password for the root user (optional)     |
 
+Following environment variables influence MySQL configuration file. They are all optional.
+
+|    Variable name         |    Description                                                  |    Default
+| :----------------------- | --------------------------------------------------------------- | -------------------------------
+|  `MYSQL_MAX_CONNECTIONS` | The maximum permitted number of simultaneous client connections |  151
+|  `MYSQL_TIME_ZONE`       | The MySQL server default timezone                               |  none
+
```

Comment:
I'd suggest reviewing [v2 mysql cart](https://github.com/openshift/origin-server/blob/master/cartridges/openshift-origin-cartridge-mysql/conf/my.cnf.erb) for variables that might be useful.